src/sage/rings/function_field/differential.py

```diff
@@ -332,26 +341,27 @@ def _acted_upon_(self, f, self_on_left):
 
         EXAMPLES::
 
-            sage: K.<x> = FunctionField(GF(31)); _.<Y> = K[]                            # optional - sage.rings.finite_rings
-            sage: L.<y> = K.extension(Y^2 - x); _.<Z> = L[]                             # optional - sage.rings.finite_rings sage.rings.function_field
-            sage: M.<z> = L.extension(Z^2 - y)                                          # optional - sage.rings.finite_rings sage.rings.function_field
-            sage: z.differential()                                                      # optional - sage.rings.finite_rings sage.rings.function_field
+            sage: # needs sage.rings.finite_rings
+            sage: K.<x> = FunctionField(GF(31)); _.<Y> = K[]
+            sage: L.<y> = K.extension(Y^2 - x); _.<Z> = L[]                             # needs sage.rings.function_field
+            sage: M.<z> = L.extension(Z^2 - y)                                          # needs sage.rings.function_field
+            sage: z.differential()                                                      # needs sage.rings.function_field
             (8/x*z) d(x)
-            sage: 1/(2*z) * y.differential()                                            # optional - sage.rings.finite_rings sage.rings.function_field
+            sage: 1/(2*z) * y.differential()                                            # needs sage.rings.function_field
             (8/x*z) d(x)
 
```

Comment:
removing this empty line should allow to avoid `needs sage.rings.finite_rings` in the next examples, no ?

Reply:
Yes, but I have instead added a block `# needs` tag